### ISSUE TITLE
The last tab is only partially highlighted after scrolling the pop up, on Windows

### ISSUE THREAD

**Issue by Softvision-CristinaBadescu** [mozilla/side-view#204]
[Affected versions]:
- Firefox 59.0.2 and up
- Side View 0.3.2

[Affected Platforms]:
- All Windows

[Prerequisites]:
- Have a clean Firefox profile with the latest Side View v0.3.2 installed.
- Have at least 21 tabs open with content.
- Have at least 1 tab with a long title.

[Steps to reproduce]:
1. Open the Firefox browser with the profile from prerequisites.
2. Click the "Open Side View" toolbar button.
3. Scroll down the pop up.
4. Hover the tab list until you reach the bottom buttons and observe the behavior.

[Expected result]:
- Each tab is highlighted while being hovered.

[Actual result]:
- The last tab is partially highlighted.

[Notes]:
- This issue is also reproducible for tabs with long titles, but in most of the cases only after you encounter the issue by performing the above steps for a tab with a short title.
- This issue is also reproducible for random tabs if you scroll and move the cursor outside the pop up area.
- This issue is not reproducible if the pop up does not contain a tab with a long title.
- This issue is not reproducible if the last tab has a title that contains one 1 word(eg. "Wikipedia").
- Here is a screen recording with this issue:
![side view - gray highlight](https://user-images.githubusercontent.com/31244235/39244888-bf77ef20-489a-11e8-8c3f-5f1fa781ac3a.gif)

**Comment by wresuolc** [comment]
duplicate of #178 

this has the same cause
